packages/dds/tree/src/feature-libraries/modular-schema/modularChangeEncoding.ts

```diff
@@ -25,6 +31,11 @@ interface EncodedNodeChangeset {
     fieldChanges?: EncodedFieldChangeMap;
 }
 
+interface EncodedModularChangeset {
+    maxId: ChangesetLocalId;
```

Comment:
Consider making the `maxId` field optional so we don't pay constant overhead for it when we're not using move.